sessions/recap-project-1/quiz-app/README.md

```diff
@@ -62,7 +73,9 @@ The second page is the bookmarks page.
 - Update the navbar so that the bookmark icon is highlighted.
 - Link the two pages via the anchor elements. (Also on the homepage).
 
-## Extra: Profile Page
+## Bonus: Profile Page
```

Comment:
```suggestion
### Bonus: Profile Page
```

Reply:
It was decided to keep the Bonus as h2.
Task are mandatory feature, meanwhile bonus and bonus, extra